core/services/chainlink/testdata/config-full.toml

```diff
@@ -335,6 +335,8 @@ TransactionPercentile = 15
 HistoryDepth = 15
 MaxBufferSize = 17
 SamplingInterval = '1h0m0s'
+MaxAllowedFinalityDepth = 1500
+FinalityTagSupportDisabled = false
```

Comment:
Yes, they should resolve the naming conflict. 
But I deliberately pick different name to avoid confusion. If HeadTracker has `FinalityTagEnabled = true`, while Chain's option is set to false, one might expect that HeadTracker will still use FinalityTags.